src/os/ObjectStore.h

```diff
@@ -872,7 +872,6 @@ class ObjectStore {
     Op* _get_next_op() {
       if (op_ptr.length() == 0 || op_ptr.offset() >= op_ptr.length()) {
         op_ptr = bufferptr(sizeof(Op) * OPS_PER_PTR);
-	op_ptr.zero();
```

Comment:
nothing the matter with this patch. It seemed that " if (op_ptr.length() == 0 || op_ptr.offset() >= op_ptr.length()) {" should be " if (op_ptr.length() == 0 || op_ptr.offset() + sizeof(Op) >= op_ptr.length()) {"
?